app/views/shop/products/_shop_variant_no_group_buy.html.haml

```diff
@@ -9,7 +9,7 @@
       -# U+FF0D Fullwidth Hyphen-Minus
       －
     %input.variant-quantity{ type: "number", min: "0", max: "{{ available() }}",
-      ng: {model: "variant.line_item.quantity", max: "Infinity"}}>
+      ng: {model: "variant.line_item.quantity", attr: { id: "variant_line_item_quantity_{{variant.line_item.id}}" }, change: "updateCart(variant.line_item.quantity)", max: "Infinity"}}>
```

Comment:
You pass in the quantity even though the method asks for the line item. Since the method ignores the parameter completely and just uses the line item of the scope, it doesn't matter but that should be fixed at some point.